benchmark_runner/oadp/oadp_workloads.py

```diff
@@ -2805,6 +2805,33 @@ def this_is_downstream(self):
         if self.__test_env['source'] != 'upstream':
             return True
 
+    @logger_time_stamp
+    def invoke_log_collection(self, scenario):
+        """
+        method invokes misc-scripts/benchmark-runner-wrapper/log_collector.sh
+        required to pass crname from tests.yaml
+        """
+
+        log_collector_sh = (f"{self.__oadp_misc_dir}/benchmark-runner-wrapper/log_collector.sh")
+        cr_name = scenario['args']['OADP_CR_NAME']
+        logger.info(f"invoke_log_collection is attempting to collect logs from cr: {cr_name} and write logs to dir: {self._run_artifacts_path}")
+        log_collection_output = self.__ssh.run(cmd=f"{log_collector_sh} {cr_name} --enable-collect-events --logs-folder {self._run_artifacts_path}")
+        logger.info(f"log_collection_output: {log_collection_output}")
+
+        logger.info(f"Getting the list of file names that exist in {self._run_artifacts_path} folder")
+        list_of_files = list(self.__ssh.run(cmd=f"find {self._run_artifacts_path} -type f").splitlines())
+        logger.info(f"{list_of_files}")
+
+        logger.info(f"Getting the size of the files in {self._run_artifacts_path} folder")
```

Comment:
once we have the list of file names that are logs, since we know the dir and now the name of the file, we can consider to just simplify the rest of the flow by using 


`os.path.exists(os.path.join(LOG FILE PATH)) and not os.stat(LOG FILE PATH)).st_size == 0`